src/main/java/org/jenkinsci/test/acceptance/selenium/Scroller.java

```diff
@@ -107,6 +109,20 @@ public void beforeClear(WebElement element) {
 
     @Override
     public void afterGet(WebDriver driver, String url) {
+        disableStickyElementsIgnoringDialogs();
+    }
+
+    @Override
+    public void afterTo(Navigation navigation, String url) {
+        disableStickyElementsIgnoringDialogs();
+    }
+
+    @Override
+    public void afterTo(Navigation navigation, URL url) {
+        disableStickyElementsIgnoringDialogs();
+    }
+
+    private void disableStickyElementsIgnoringDialogs() {
```

Comment:
`disableStickyElements` is public and called in the suite, so introduce this rather than changing the public method for now